asreviewcontrib/dory/feature_extractors/transformer_embeddings.py

```diff
@@ -46,13 +94,24 @@ def __init__(
                 "sentence_transformer",
                 BaseSentenceTransformer(
                     model_name=self.model_name,
-                    normalize=self.normalize,
-                    quantize=self.quantize,
-                    precision=self.precision,
                     verbose=self.verbose,
                 ),
             ),
         ]
+        if self.normalize:
+            if self.normalize_method == "l2":
```

Comment:
I like the overall design now, except for this. In general, I would expect [None, l2, minmax, standard]. In that case, you don't need two variables to achieve the same thing. But maybe you do this for backward compat.

Reply:
Yes indeed, it's to not have to change the h3/l2 models (normalize==True with normalize_method's default is the same as it was). I can also make a PR to change h3/l2's parameters, I would prefer that tbh, to keep the API of Dory neat.